package-lock.json

```diff
@@ -7495,7 +7230,7 @@
         "string-width": "^3.0.0",
         "which-module": "^2.0.0",
         "y18n": "^4.0.0",
-        "yargs-parser": "^13.1.1"
+        "yargs-parser": "^13.1.2"
```

Comment:
Can you make sure package-lock does not appear in this diff?

Reply:
I have reverted yargs-parser back to 13.1.1 and commited the new version - that's what you wanted right? Or did you want me to use the same package-lock.json file entirely?

Reply:
Yes, package-lock should be as is and not be shown in the diff. No need to upgrade any module as part of this PR.